src/js/support/mobile/widget/ToggleSwitch.extra.js

```diff
@@ -534,9 +534,6 @@
 
 					// If changes came from input value change
 				} else {
-					if (value === null) {
```

Comment:
I'm not sure this change. We probably don't take account "undefined" value, but before also didn't take account it.

Reply:
Yes, undefined was not handled and adding code for this may cause different behavior.

Reply:
Ok. Looks good to me.